src/lib/features/traffic-data-usage/fake-traffic-data-usage-store.ts

```diff
@@ -89,4 +89,41 @@ export class FakeTrafficDataUsageStore implements ITrafficDataUsageStore {
 
         return Object.values(data);
     }
+
+    async getDailyTrafficDataForPeriod(
+        from: Date,
+        to: Date,
+    ): Promise<IStatTrafficUsage[]> {
+        return this.trafficData.filter(
+            (data) => data.day >= from && data.day <= to,
```

Comment:
from/to, will they contain time? day field in DB is day, so might potentially catch one less day than intentional.

Reply:
This is just the fake store, so I'm not too worried about this one, but yeah, we should add a "startOfday/endOfDay here too. I'll fix.

Reply:
ah good point, dunno why I missed that 😅

Reply:
addressed in 9a5863b7b04015c6ebbb25083665e72f97a4c455